### PR TITLE
Separate out common nginx config

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ services:
        - .env
     volumes:
       - ./proxy-template.conf:/etc/nginx/templates/default.conf.template:ro
+      - ./common.conf:/etc/nginx/templates/common.conf.template:ro
```
Comment (thenav56) [inline docker-compose.yml]: Lets create a subdirectory with common and nginx template file and then we can use /etc/nginx/templates/ to pass all the files with one volut

File: common.conf
Comment (thenav56) [inline common.conf:1]: let's use templates/proxy.conf

**Review by thenav56 — CHANGES_REQUESTED** [review]
Few suggestions

File: docker-compose.yml (hunk 1 above)
Comment (tnagorra) [reply docker-compose.yml]: I agree.

File: .env.template [hunk 2]
```diff
@@ -1,6 +1,6 @@
-SOURCE_PROTOCOL=https
-SOURCE_DOMAIN=subdomain.example.com
-DESTINATION_PORT=8001
+SOURCE1_PROTOCOL=https
+SOURCE1_DOMAIN=subdomain.example.com
+DESTINATION1_PORT=8001
```
Comment (thenav56) [inline .env.template]: I think SOURCE_PROTOCOL is fine here,

User can modify the `templates/default.conf.template` to support multiple domain using `/etc/hosts` where they can then use `server_name` and `/etc/hosts` to support multiple proxies.

Comment (thenav56) [reply .env.template]: Maybe we can also push this changes in a different branch specifically for idmc-website-components

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: .env.template (hunk 2 above)
Comment (tnagorra) [reply .env.template]: I was thinking we could just create a fork on the idmc workspace

**Review by thenav56 — APPROVED** [review]

**Pull request merged**